multiphysics/unsteady_fsi_python/ObtainSU2Results.m

```diff
@@ -0,0 +1,114 @@
+%% \file ObtainSU2Results.m
+%  \brief Retrives the SU2-Nastran results

```

Comment:
There is a strange symbol here?

Reply:
Very strange, I just have no idea what that is ... removing

Reply:
Ok I know what that is... it is a hidden symbol that Matlab uses for the title line of a function. It is added when you place the double %% at the beginning. If you open the file in Matlab it disappears

Reply:
`^M` is a carriage return. I get rid of these using vim on linux

*Conversely, when viewing a file originating from a Windows computer on a Unix-like system, the extra CR may be displayed as a second line break, as ^M, or as <cr> at the end of each line. * see [here](https://en.wikipedia.org/wiki/Newline) ...

Reply:
Ok thanks, I will remove them